sqlite/sqlite_helpers.go

```diff
@@ -0,0 +1,29 @@
+package sqlite
```

Comment:
This code was moved from `sqlite_test` so that the `NewTestStore` method could be used in other packages, like the `notebooks` package tests.